src/server/routes/index.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express');
+var router = express.Router();
+var config = require('../config');
+var _ = require('lodash');
+
+router.get('/config', function (req, res, next) {
+  var excludedKeys = [
+    'port',
+    'host',
+    'verify_ssl',
+    'request_timeout',
+    'elasticsearch_url',
+    'elasticsearch_username',
+    'elasticsearch_password',
+    'elasticsearch_preserve_host',
+    'bundled_plugin_ids'
+  ];
+  var data = _.omit(config.kibana, excludedKeys);
```

Comment:
Should we just use `pick` here? Doesn't seem like there are my config vars you actually care about:

![screenshot 2015-01-21 13 09 49](https://cloud.githubusercontent.com/assets/404731/5844146/e6b9dd72-a16e-11e4-84fe-5a3df4a75888.png)